public/js/functions.js

```diff
@@ -18,7 +18,12 @@ function mixin(obj, mix) {
 
 function Sum(list) {
 	var sum = 0;
-	list.forEach(function(v,i) { if (v !== undefined) sum += v; });
+	if(list===undefined) return sum;
```

Comment:
Leave this in for now. Long term we should change all models to use ifisnan to convert the nan's to 0.